src/components/admin/Toggle/Toggle.tsx

```diff
@@ -39,7 +39,7 @@ export const Toggle: React.FC<ToggleProps> = ({
   const trackClasses = TW[`${!!checked}Track`];
 
   return (
-    <div className="flex flex-col w-full mb-3">
+    <>
       {title && (
         <label className="block text-sm font-medium text-gray-700">
```

Comment:
[OOS] This still doesn't look right. there is no id connecting this label with the input later on.